RELEASE_NOTES.md

```diff
@@ -23,6 +24,7 @@
 ### Changes
 - Podman now passes container hostnames to Netavark, which will use them for any DHCP requests for the container.
 - Partial pulls of `zstd:chunked` images now only happen for images that have a `RootFS.DiffID` entry in the image's OCI config JSON, and require the layer contents to match. This resolves issues with image ID ambiguity when partial pulls were enabled.
+- Packagers can now set the `BUILT_FOR` environment variable when building podman from the `Makefile`. This provides information on who built the Podman binary, and is displayed in `podman version` and  `podman info`. This will help upstream bug reports, allowing maintainers to trace how and where the binary was built and installed from.
```

Comment:
BUILT_FOR is wrong, look at the diff it is using BUILD_ORIGIN in the Makefile

Also not here of course but we really should set this in our rpm specfile here as well.

Reply:
can `BUILD_ORIGIN` be any random string?

I guess we could get this in for v5.4.0 final.

Reply:
ah ok, just tried it. `Build Origin: foo` . For rpm, we could set it to something like `Build Origin: rhcontainerbot/podman-next` and `Build Origin: Koji` for copr v/s koji rpms.

Reply:
yes something like that, not sure if koji is good name as I don't think many know what koji is.
I was thinking in lines of `fedora-offical`, `centos-offical`, `rhel-offical` or something like it so we know it is from the official repos. Then maybe other official/main distro packages can match that we could have some consistency at least, e.g. `debian-offical`, `arch-offical`.
Using `rhcontainerbot/podman-next` sounds like a good thing for the copr builds there.

Reply:
WIP at rhttps://github.com/containers/podman/pull/25235 . We're not using Makefile for building the binaries, so I'm setting LDFLAGS separately in the spec file.